certification/internal/policy/operator/operator_pkg_name_uniqueness.go

```diff
@@ -11,14 +11,6 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const (
```

Comment:
This belongs in a separate PR.

Reply:
The DeployableByOLM requires the package key, among the others. The PackageUniqueness check also requires this key. I was not comfortable using a const which has been defined in another file, and that was the main motivation to move all these common constants into a central place.

Reply:
Ok. Perfect. I didn't gleam immediately that it was related. Totally fine with that.